.github/how_to_compile_gudhi_in_a_conda_env.md

```diff
@@ -0,0 +1,97 @@
+# Install a conda development environment to compile GUDHI
+
+## Install miniconda
+
+Download the [installer](https://docs.conda.io/en/latest/miniconda.html) required by your system and follow the [instructions](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
```

Comment:
I used miniforge last time, it uses 100% conda-forge (anaconda isn't even configured). I am just listing it here, no need to change.